etl/src/types/mod.rs

```diff
@@ -0,0 +1,14 @@
+mod pipeline;
+
+// TODO: properly implement types in this module and export them.
+pub use crate::conversions::{
```

Comment:
I just hid behind the `types` module all the exported types in `etl`, but this will also have to be done in a better way by actually splitting types from conversions.